tests/test_utils.py

```diff
@@ -62,3 +62,23 @@ def test_get_workspace_uri_request_exception(mock_get):
     TestCase.assertRaises(
         TestCase, requests.exceptions.RequestException, get_workspace_uri
     )
+
+
+@mock.patch("builtins.open")
+@mock.patch("json.load", mock.MagicMock(side_effect=[{"test": "test"}]))
+def test_read_json_successful(*args):
+    """
+    Tests when file is found
+    """
+    response = read_json("path/to/open")
+    assert response == {"test": "test"}
+    assert response != None
+
+
+@mock.patch("builtins.open", side_effect=FileNotFoundError)
+def test_read_json_file_not_found(mock_open):
+    """
+    Tests when the JSON file is not found.
+    """
+    with TestCase.assertRaises(TestCase, FileNotFoundError):
```

Comment:
Please use `pytest.raises` ([docs](https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions)):

```python
with pytest.raises(FileNotFoundError):
...
```

If you could also make this change for line 67 it will be greatly appreciated